core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -0,0 +1,241 @@
+
+                    <br>
+                    <br>
+                    <br>
+
+                    Status:
+                    <span ng-if="collection.status !== 'private'">
+                      <span class="oppia-dashboard-status-green">
+                        Published
+                      </span>
+                    </span>
+                    <span ng-if="collection.status === 'private'">
+                      <span class="oppia-dashboard-status-grey">
+                        Private
+                      </span>
+                    </span>
+                  </div>
+                </div>
+                <div class="oppia-dashboard-tile-contents">
+                  <span class="oppia-dashboard-tile-image-container" style="position: absolute;">
+                    <div class="oppia-dashboard-tile-collection-label">Collection</div>
+                    <img class="oppia-dashboard-tile-image" ng-src="<[collection.thumbnail_icon_url]>" style="background-color: <[collection.thumbnail_bg_color]>" ng-class="{'oppia-dashboard-tile-image-private': collection.status === 'private'}">
+                  </span>
+
+                  <div class="oppia-dashboard-tile-details">
+                    <div class="oppia-dashboard-tile-first-row">
+                      <span class="oppia-dashboard-tile-title">
+                        <span ng-if="collection.title"><[collection.title]></span>
+                        <span ng-if="!collection.title">Untitled</span>
+                      </span>
+                      <span style="font-weight: bold;">&middot;</span>
+                      <span style="color: #888; font-size: 0.9em;">
+                        <span ng-if="collection.objective">
+                          <[collection.objective | truncateAndCapitalize: 105]>
+                        </span>
+                        <span ng-if="!collection.objective">
+                          <em>No objective specified.</em>
+                        </span>
+                      </span>
+                    </div>
+
+                    <div class="oppia-dashboard-tile-second-row">
+                      <span style="color: #888">
+                        <strong><[collection.category]></strong>
+                        <span style="font-weight: bold;" ng-if="collection.category">&middot;</span>
+                        Last updated <[getLocaleAbbreviatedDatetimeString(collection.last_updated)]>
+                      </span>
+
+                      <span ng-if="collection.status === 'publicized'">
+                        <span style="font-weight: bold;">&middot;</span>
+                        <span style="color: #888">Featured</span>
+                      </span>
+                    </div>
+                  </div>
+                </div>
+              </a>
+            </md-card>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div ng-if="explorationsList.length === 0">
```

Comment:
Should this whole bit only show up on the 'My Explorations' tab and take the place of the current ``There are no explorations to display``?